lib/pages/main/main_vc.dart

```diff
@@ -79,6 +82,20 @@ class _MainVCState extends State<MainVC> {
     }
   }
 
+  void _getNotificationCount() async {
+    List<MyNotification> notifications = await ApiSingleton().getNotifications();
+    var unacknowledgedCount = notifications.where((notification) => notification.acknowledged == false).length;
+    setState(() {
+      unreadNotifications = unacknowledgedCount;
+    });
```

Comment:
What do you think about using `updateNotificationCount(unacknowledgedCount)` , just for the sake of avoiding duplicating code..